diff -up tg_owt-62321fd7128ab2650b459d4195781af8185e46b5/cmake/external.cmake.2~ tg_owt-62321fd7128ab2650b459d4195781af8185e46b5/cmake/external.cmake
--- tg_owt-62321fd7128ab2650b459d4195781af8185e46b5/cmake/external.cmake.2~	2025-06-03 07:20:04.000000000 +0200
+++ tg_owt-62321fd7128ab2650b459d4195781af8185e46b5/cmake/external.cmake	2025-07-26 01:02:09.891293951 +0200
@@ -87,8 +87,8 @@ function(link_libabsl target_name)
     endif()
     if (TG_OWT_PACKAGED_BUILD)
         find_package(absl)
-        set(absl_FOUND ${absl_FOUND} PARENT_SCOPE)
-        if (absl_FOUND)
+	set(absl_FOUND TRUE PARENT_SCOPE)
+	if (TRUE)
             target_link_libraries(${target_name}
             ${scope}
                 absl::algorithm_container
