diff -up tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/CMakeLists.txt.1~ tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/CMakeLists.txt
--- tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/CMakeLists.txt.1~	2024-07-29 20:39:28.445800438 +0200
+++ tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/CMakeLists.txt	2024-07-29 20:39:43.190033150 +0200
@@ -74,7 +74,7 @@ include(cmake/init_target.cmake)
 include(cmake/generate_target.cmake)
 include(cmake/generate_field_trials.cmake)
 include(cmake/external.cmake)
-include(cmake/libabsl.cmake)
+#include(cmake/libabsl.cmake)
 include(cmake/libcrc32c.cmake)
 include(cmake/libpffft.cmake)
 include(cmake/librnnoise.cmake)
diff -up tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/cmake/external.cmake.1~ tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/cmake/external.cmake
--- tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/cmake/external.cmake.1~	2024-07-29 15:28:29.000000000 +0200
+++ tg_owt-48c9c31d591509799a8385542ff3fb04f4d58327/cmake/external.cmake	2024-07-29 20:39:28.445800438 +0200
@@ -100,8 +100,8 @@ endfunction()
 function(link_libabsl target_name)
     if (TG_OWT_PACKAGED_BUILD)
         find_package(absl)
-        set(absl_FOUND ${absl_FOUND} PARENT_SCOPE)
-        if (absl_FOUND)
+	set(absl_FOUND TRUE PARENT_SCOPE)
+	if (TRUE)
             target_link_libraries(${target_name}
             PUBLIC
                 absl::algorithm_container
