diff -up tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt.3~ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt
--- tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt.3~	2022-07-12 03:05:17.714436369 +0200
+++ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt	2022-07-12 03:05:37.073955642 +0200
@@ -804,8 +804,6 @@ PRIVATE
     media/engine/webrtc_media_engine_defaults.cc
     media/engine/webrtc_video_engine.cc
     media/engine/webrtc_voice_engine.cc
-    media/sctp/dcsctp_transport.cc
-    media/sctp/dcsctp_transport.h
     media/sctp/sctp_transport_factory.cc
     media/sctp/sctp_transport_factory.h
     system_wrappers/include/denormal_disabler.h
