diff -up tg_owt-d5c3d43b959c7e9e7d8004b9b7fdadd12ce7d589/CMakeLists.txt.3~ tg_owt-d5c3d43b959c7e9e7d8004b9b7fdadd12ce7d589/CMakeLists.txt
--- tg_owt-d5c3d43b959c7e9e7d8004b9b7fdadd12ce7d589/CMakeLists.txt.3~	2021-12-13 16:10:48.757048284 +0100
+++ tg_owt-d5c3d43b959c7e9e7d8004b9b7fdadd12ce7d589/CMakeLists.txt	2021-12-13 21:24:43.838850717 +0100
@@ -801,6 +801,71 @@ PRIVATE
     media/sctp/sctp_transport_factory.h
     media/sctp/usrsctp_transport.cc
     media/sctp/usrsctp_transport.h
+    net/dcsctp/fuzzers/dcsctp_fuzzers.cc
+    net/dcsctp/timer/timer.cc
+    net/dcsctp/timer/task_queue_timeout.cc
+    net/dcsctp/testing/data_generator.cc
+    net/dcsctp/socket/dcsctp_socket.cc
+    net/dcsctp/socket/heartbeat_handler.cc
+    net/dcsctp/socket/transmission_control_block.cc
+    net/dcsctp/socket/state_cookie.cc
+    net/dcsctp/socket/stream_reset_handler.cc
+    net/dcsctp/tx/retransmission_queue.cc
+    net/dcsctp/tx/retransmission_timeout.cc
+    net/dcsctp/tx/retransmission_error_counter.cc
+    net/dcsctp/tx/rr_send_queue.cc
+    net/dcsctp/packet/crc32c.cc
+    net/dcsctp/packet/error_cause/stale_cookie_error_cause.cc
+    net/dcsctp/packet/error_cause/unresolvable_address_cause.cc
+    net/dcsctp/packet/error_cause/no_user_data_cause.cc
+    net/dcsctp/packet/error_cause/invalid_mandatory_parameter_cause.cc
+    net/dcsctp/packet/error_cause/missing_mandatory_parameter_cause.cc
+    net/dcsctp/packet/error_cause/unrecognized_chunk_type_cause.cc
+    net/dcsctp/packet/error_cause/protocol_violation_cause.cc
+    net/dcsctp/packet/error_cause/out_of_resource_error_cause.cc
+    net/dcsctp/packet/error_cause/cookie_received_while_shutting_down_cause.cc
+    net/dcsctp/packet/error_cause/invalid_stream_identifier_cause.cc
+    net/dcsctp/packet/error_cause/restart_of_an_association_with_new_address_cause.cc
+    net/dcsctp/packet/error_cause/unrecognized_parameter_cause.cc
+    net/dcsctp/packet/error_cause/user_initiated_abort_cause.cc
+    net/dcsctp/packet/error_cause/error_cause.cc
+    net/dcsctp/packet/parameter/add_outgoing_streams_request_parameter.cc
+    net/dcsctp/packet/parameter/forward_tsn_supported_parameter.cc
+    net/dcsctp/packet/parameter/add_incoming_streams_request_parameter.cc
+    net/dcsctp/packet/parameter/parameter.cc
+    net/dcsctp/packet/parameter/heartbeat_info_parameter.cc
+    net/dcsctp/packet/parameter/incoming_ssn_reset_request_parameter.cc
+    net/dcsctp/packet/parameter/state_cookie_parameter.cc
+    net/dcsctp/packet/parameter/ssn_tsn_reset_request_parameter.cc
+    net/dcsctp/packet/parameter/supported_extensions_parameter.cc
+    net/dcsctp/packet/parameter/outgoing_ssn_reset_request_parameter.cc
+    net/dcsctp/packet/parameter/reconfiguration_response_parameter.cc
+    net/dcsctp/packet/sctp_packet.cc
+    net/dcsctp/packet/tlv_trait.cc
+    net/dcsctp/packet/chunk_validators.cc
+    net/dcsctp/packet/chunk/shutdown_ack_chunk.cc
+    net/dcsctp/packet/chunk/iforward_tsn_chunk.cc
+    net/dcsctp/packet/chunk/chunk.cc
+    net/dcsctp/packet/chunk/sack_chunk.cc
+    net/dcsctp/packet/chunk/abort_chunk.cc
+    net/dcsctp/packet/chunk/idata_chunk.cc
+    net/dcsctp/packet/chunk/cookie_echo_chunk.cc
+    net/dcsctp/packet/chunk/cookie_ack_chunk.cc
+    net/dcsctp/packet/chunk/forward_tsn_chunk.cc
+    net/dcsctp/packet/chunk/reconfig_chunk.cc
+    net/dcsctp/packet/chunk/heartbeat_request_chunk.cc
+    net/dcsctp/packet/chunk/heartbeat_ack_chunk.cc
+    net/dcsctp/packet/chunk/shutdown_complete_chunk.cc
+    net/dcsctp/packet/chunk/shutdown_chunk.cc
+    net/dcsctp/packet/chunk/init_chunk.cc
+    net/dcsctp/packet/chunk/data_chunk.cc
+    net/dcsctp/packet/chunk/init_ack_chunk.cc
+    net/dcsctp/packet/chunk/error_chunk.cc
+    net/dcsctp/public/text_pcap_packet_observer.cc
+    net/dcsctp/public/dcsctp_socket_factory.cc
+    net/dcsctp/rx/reassembly_queue.cc
+    net/dcsctp/rx/traditional_reassembly_streams.cc
+    net/dcsctp/rx/data_tracker.cc
     system_wrappers/include/denormal_disabler.h
     system_wrappers/source/clock.cc
     system_wrappers/source/cpu_features.cc
@@ -810,6 +875,7 @@ PRIVATE
     system_wrappers/source/metrics.cc
     system_wrappers/source/rtp_to_ntp_estimator.cc
     system_wrappers/source/sleep.cc
+    third_party/crc32c/src/src/crc32c.cc
     modules/audio_coding/acm2/acm_receiver.cc
     modules/audio_coding/acm2/acm_remixing.cc
     modules/audio_coding/acm2/acm_resampler.cc
