diff -up tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/cmake/external.cmake.4~ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/cmake/external.cmake
--- tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/cmake/external.cmake.4~	2022-05-08 06:44:06.000000000 +0200
+++ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/cmake/external.cmake	2022-07-12 03:05:52.380575572 +0200
@@ -102,8 +102,8 @@ endfunction()
 function(link_libabsl target_name)
     if (TG_OWT_PACKAGED_BUILD)
         find_package(absl)
-        set(absl_FOUND ${absl_FOUND} PARENT_SCOPE)
-        if (absl_FOUND)
+	set(absl_FOUND TRUE PARENT_SCOPE)
+	if (TRUE)
             target_link_libraries(${target_name}
             INTERFACE
                 absl::algorithm_container
diff -up tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt.4~ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt
--- tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt.4~	2022-07-12 03:05:52.381575548 +0200
+++ tg_owt-10d5f4bf77333ef6b43516f90d2ce13273255f41/CMakeLists.txt	2022-07-12 03:06:09.499150530 +0200
@@ -77,7 +77,7 @@ include(cmake/init_target.cmake)
 include(cmake/generate_target.cmake)
 include(cmake/target_yasm_sources.cmake)
 include(cmake/external.cmake)
-include(cmake/libabsl.cmake)
+#include(cmake/libabsl.cmake)
 include(cmake/libcrc32c.cmake)
 include(cmake/libpffft.cmake)
 include(cmake/librnnoise.cmake)
